### PR TITLE
Updates to address CVEs

### PR DESCRIPTION
**Pull request by rajeshthuraiyur**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/RIA-6885

### Change description ###
Updates to address CVEs

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X ] No
```

### CODE REVIEW

**Review by kalachandrasekar1 — DISMISSED** [review]

**Review by kalachandrasekar1 — DISMISSED** [review]

**Review by kalachandrasekar1 — APPROVED** [review]

**Review by javimarinc96 — DISMISSED** [review]
LGTM

**Review by filippo-possenti-version1 — DISMISSED** [review]

**Review by filippo-possenti-version1 — APPROVED** [review]

**Review by mcareyma — APPROVED** [review]
Magician, you managed to convince everything that uses json.smart to use at least 2.4.10

File: charts/ia-case-api/values.preview.template.yaml [hunk 1]
```diff
@@ -8,23 +8,19 @@ java:
     POSTGRES_USERNAME: hmcts
     POSTGRES_PASSWORD: hmcts
     POSTGRES_CONNECTION_OPTIONS: ""
-    #Un comment APIs if you need to test a local version
```
Comment (rajeshthuraiyur) [inline charts/ia-case-api/values.preview.template.yaml:11]: Can we put those commented lines back? its actually for reference, when we extend the preview env for other components, we need to enable those links to overwrite what is set as part of jenkins

Comment (filippo-possenti-version1) [reply charts/ia-case-api/values.preview.template.yaml:11]: Agreed.

**Review by neha-aggarw — APPROVED** [review]

**Review by neha-aggarw — APPROVED** [review]

**Review by filippo-possenti-version1 — APPROVED** [review]

**Pull request merged**